docs/tutorials/create-your-first-substrate-chain/setup.md

```diff
@@ -64,3 +66,4 @@ git clone -b v3.0.0 --depth 1 https://github.com/substrate-developer-hub/substra
 cd substrate-front-end-template
 yarn install
 ```
+> **Run into issues? Try the [troubleshooting tips](../../knowledgebase/getting-started/index#troubleshooting-substrate-builds)**
```

Comment:
Based on the assumption that the person sees the first occurrence of this link and would have already discovered its contents if they ran into an issue, IMO there's no need to duplicate this phrase. 
```suggestion
```

Reply:
They may get new error in the front end template. So it is a good idea to repeat... Maybe? And add more in a further PR on fronted issues

Reply:
I see what you're saying but there's presently nothing in the "Troubleshooting Tips" that are specific to front-end node template troubleshooting so it feels misleading to include. I think that once we write that section it would make more sense to me to link to it after the fact.